policyengine_us/variables/README.md

```diff
@@ -0,0 +1,24 @@
+## Variables
+
+- Update the following files to add the state model into income tree:
```

Comment:
I dont think we need this any more - all state model are complete

Reply:
We should rather highlight that each new program should be added to the net income tree to have an impact on the microsim / household